### PR TITLE
code climate set-up

### PR DESCRIPTION
**Pull request by cauld19**
# Code climate set-up

- github/workflows/ci.yml added
-coverage and jest collectCoverageFrom added to package.json

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by cauld19** [comment]
seems like its going through netlify still and not aws. but the code coverage is working

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  1.17.3
+
+Node version: 
+  10.16.0
+
+Platform: 
+  win32 x64
+
+Trace: 
+  SyntaxError: C:\Users\christian\booked-up-fe\package.json: Unexpected token < in JSON at position 1280
```
Comment (nathan-loveless) [inline yarn-error.log:17]: What is this error for?  Is that what you fixed in later commits?

Comment (cauld19) [reply yarn-error.log:17]: "jest": {
    "collectCoverageFrom": [
      "**/*.{js,jsx}", _<--- If you don't have .jsx files in your project this line should look like this._ "**/*.js",
      "!**/node_modules/**",
      "!**/vendor/**",
      "!**/__tests__/**"
    ]
  }

I forgot to remove the text with the arrow from above code

Comment (nathan-loveless) [reply yarn-error.log:17]: Ok sounds good, resolving

**Pull request merged**